x/ecosystem-incentive/spec/04_hooks.md

```diff
@@ -0,0 +1,43 @@
+# Hooks
+
+**NOTE: This is early draft.**
+
+All rewards accumulation are executed when the according hooks function is called.   
+
+The example hooks functions interfaces in x/nftmarket module:
+
+```go
+type NftmarketHooks interface {
+   AfterNftListed(ctx sdk.Context, nft_id types.NftIdentifier, incentive_id string)
```

Comment:
Put tx.memo in the argument.
incentive_id is not required
```
before
AfterNftListed(ctx sdk.Context, nft_id types.NftIdentifier, incentive_id string)
after
AfterNftListed(ctx sdk.Context, nft_id types.NftIdentifier, tx_memo string)
```

Reply:
Modified as it shows.
https://github.com/UnUniFi/chain/blob/23b8c63c40c112e6adceb0c40ae492c5ba8067b5/x/ecosystem-incentive/spec/04_hooks.md?plain=1#L11